python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -91,6 +91,39 @@ def createButler(self):
 
         return butler
 
+    def extract_info(self, dataId, key):
+        if key in dataId:
+            return f"{dataId[key]}"
+        return "??"
+
+    def extract_info_val(self, dataId, key, prefix):
+        if key in dataId:
+            return f"{prefix}%.2d" % dataId[key]
```

Comment:
This seems really strained — mixing f-strings and `%` formatting.  You can format f-strings as well:
```
return f"{prefix}{dataId[key]:02d}"
```

Reply:
Ah - thanks.  Wasn't sure how to do that.  Fixed.